zcash_test_vectors/zip_0244.py

```diff
@@ -156,25 +162,28 @@ def orchard_auth_digest(tx):
 
 def orchard_actions_compact_digest(tx):
     digest = blake2b(digest_size=32, person=b'ZTxIdOrcActCHash')
+    offset = 32 if tx.nVersionGroupId == NU7_VERSION_GROUP_ID else 0
```

Comment:
I'd prefer naming it something other than `compact_offset` and `memo_offset`, because the offset for both is 0 or 32.

- 52 / 84 is the end of the compact note.
- 564 / 596 is the end of the memo field (which is the same size, just starts later due to the compact note being larger).

Would `compact_end` and `memo_end` be alright?

Reply:
I prefer using just a single `offset`. We could structure it better by generating the value in a `ciphertext_offset(tx_version)` function as suggested.

Reply:
compact_end and memo_end are good names. We still need a function.

Reply:
Done